x/stake/keeper/validator.go

```diff
@@ -88,6 +116,37 @@ func (k Keeper) GetValidators(ctx sdk.Context, maxRetrieve ...int16) (validators
 	return validators[:i] // trim
 }
 
+// Get the set of all validators, retrieve an optional maxRetrieve number of records
+func (k Keeper) GetBechValidators(ctx sdk.Context, maxRetrieve ...int16) (validators []types.BechValidator) {
+	retrieve := len(maxRetrieve) > 0
```

Comment:
Seems we can DRY up the `GetBechValidators` and `GetDelegatorBechValidators` methods that do iteration by utilizing the existing non-bech methods and calling `Bech32Validator` on the results.

e.g.
```golang
func (k Keeper) GetBechValidators(ctx sdk.Context, maxRetrieve ...int16) ([]types.BechValidator) {
    validators := k.GetValidators(ctx, maxRetrieve...)
    bechValidators := make([]types.BechValidator, len(validators)
    
    for i, val := range validators {
        bechValidator, err := validator.Bech32Validator()
	if err != nil {
		panic(err.Error())
	}

        bechValidators[i] = bechValidator
    } 

    return bechValidators
}
```

Does this make sense?

Reply:
yeah but that would iterate over the validators twice, one for `k.GetValidators` and another for `GetBechValidators`